### ISSUE TITLE
No network connection available, aborting provision.

### ISSUE THREAD

**Issue by mrigankvallabh** [Varying-Vagrant-Vagrants/VVV#1647]
## Expected Behavior
VVV should provision local worpress environment

## Current Behavior
Provisioning fails

## Steps to Reproduce (for bugs)
This problem is report already in issue 1585.

https://gist.github.com/mrigankvallabh/e8f6fefe21c817c384cb8b8f1e8f0c69

## Your Environment (All latest version)
* VVV version: 2.3.0
* VVV Git Branch: master
* Vagrant version: 2.1.5
* VM Provider name: VirtualBox
* VM Provider version: 5.2.18
* Operating System and version: Windows 10

**Comment by mrigankvallabh** [comment]
/* Update */

The problem only happens on master branch. I cloned the 'develop' branch and this problem does not occur. I tried on a different computer and I faced the problem again on Master branch.

**Comment by tomjn** [comment]
hmm that's suspicious, we not long released v2.3, so `master` and `develop` are near identical. There's nothing to cause such a difference

Perhaps the VM is just broken, running `vagrant destroy && vagrant up --provision` to destroy it and rebuild it might be what you need to do. I also suspect that provisioning a new VVV on the `master` branch in a different folder would work for the same reason it worked with the `develop` branch

**Comment by tomjn** [comment]
Updating the VB Guest additions via the vagrant plugin `vagrant-vbguest` may also help, as might `vagrant box update`, but keep in mind that if you update the box, or do the destroy/reload, you might loose your databases. Check that the DB backups have been made first

**Comment by mrigankvallabh** [comment]
I have already tried vagrant destroy && vagrant up --provision many times. Not only that I also tried with master branch on two different computers and it fails for exactly the same reason. Even I am confused as to why this would be happening. I will once again try to provision in a new folder using vagrant-vbguest. Anyway this is my first time experience with Virtual Machines, so I can experiment a lot, as I am learning wordpress theme development as a hobby. By the way I must also say that having installed the develop branch has given me a working wordpress environment.
